resolwe_bio/processes/variant_calling/gatk_variant_refinement.py

```diff
@@ -0,0 +1,126 @@
+"""Run GATK genotype refinement."""
+
+import os
+from pathlib import Path
+
+from plumbum import TEE
+
+from resolwe.process import (
+    Cmd,
+    DataField,
+    FileField,
+    Process,
+    SchedulingClass,
+    StringField,
+)
+
+
+class GatkGenotypeRefinement(Process):
+    """Run GATK Genotype Refinement."""
+
+    slug = "gatk-refine-variants"
+    name = "GATK refine variants"
+    category = "GATK"
+    process_type = "data:variants:variants:refinevariants"
+    version = "1.0.0"
+    scheduling_class = SchedulingClass.BATCH
+    entity = {"type": "sample"}
+    requirements = {
+        "expression-engine": "jinja",
+        "executor": {
+            "docker": {"image": "public.ecr.aws/genialis/resolwebio/dnaseq:6.1.0"}
+        },
+        "resources": {
+            "cores": 2,
+            "memory": 16384,
+        },
+    }
+    data_name = '{{ vcf|sample_name|default("?") }}'
+
+    class Input:
+        """Input fields for GatkGenotypeRefinement."""
+
+        vcf_main = DataField(
+            "variants:vcf", label="The main input, as produced in the gatk-vqsr process"
+        )
+        ref_seq = DataField("seq:nucleotide", label="Reference sequence")
+
+        vcf_pop = DataField(
+            "variants:vcf",
+            label="Population-level variant set VCF data",
+            required=False,
+        )
+
+        ped = FileField(label="PED file describing pedigree", required=False)
+
+    class Output:
+        """Output fields for GatkGenotypeRefinement."""
+
+        vcf = FileField(label="Refined multi-sample vcf")
+        tbi = FileField(label="Tabix index")
+        species = StringField(label="Species")
+        build = StringField(label="Build")
+
+    def run(self, inputs, outputs):
```

Comment:
Add an additional check to the body of the process which validates that the `species` and `build` info for the `ref_seq` and `vcf` inputs are the same. If they are not, fail the process with a warning. If the `vcf_pop` is given, check the same also for this input.